src/main/java/net/dv8tion/jda/api/interactions/commands/build/CommandData.java

```diff
@@ -51,37 +50,38 @@ public interface CommandData extends SerializableData
     CommandData setName(@Nonnull String name);
 
     /**
-     * Whether this command is available to everyone by default.
-     * <br>If this is disabled, you need to explicitly whitelist users and roles per guild.
+     * Sets the default {@link CommandPermissions} for this command.
```

Comment:
The usage of "default" makes it very confusing since you also call `ENABLED` "default", this could be more clear.

Reply:
I updated those docs abit now. Not sure whether that makes it better though.